apptail/instance.go

```diff
@@ -37,7 +38,7 @@ func (instance *Instance) Identifier() string {
 }
 
 // Tail begins tailing the files for this instance.
-func (instance *Instance) Tail() {
+func (instance *Instance) Tail(tracker storage.Tracker) {
```

Comment:
That would change the singleton behaviour of the tracker though, since instance.Tail is being called per nats subscription in an anonymous way.
```
	natsclient.Subscribe("logyard."+uid+".newinstance", func(instance *apptail.Instance) {
		instance.Tail(tracker)
	})
```